HTML/Forget Maze Not.html

```diff
@@ -17,25 +18,25 @@
-                            <li>Blue: Same as yellow, but instead of inputting the walls, input the stage number the cell was displayed on using the arrows.</li>
-                            <li>Magenta: The defuser will have to find the exit of a normal 5x5 maze on the module.</li>
-                            <li>Cyan: Same as magenta, but the maze is inverted (You can only walk through walls).</li>
-                        </ul>
-                    </li>
-                    <li>Moving, exiting interactive mode, and submitting cells incorrecty will lead to a strike.</li>
-                </ul>
-                <img src="img/Forget Maze Not/Maze.svg" width="300" height="300" style="display:block;margin-left:auto;margin-right:auto;margin-top:20px;">
+				<img src="img/Component/Forget Maze Not.svg" class="diagram">
+				<h2>On the Subject of Forget Maze Not</h2>
+				<p class="flavour-text">flavor in get you I hope didn't lost this text!</p>
+				<ul>
+					<li>This module will cycle through a few cells of a maze, showing the walls and coordinates of the cell.</li>
+					<li>Coordinates are in LetterNumber format; A1 is top left.</li>
+					<li>With every solve, ignoring some specific modules, the module will move to the next stage, revealing more cells. Put them together to form a maze.</li>
+					<li>On some stages, a cell will be colored green. After all the non-ignored modules are solved, navigate to these cells in the order of their appearance and submit them by clicking the middle button.</li>
+					<li>Any other colored cell are special cells and they are not part of the main maze. Look at the direction's below for the respective color. Click the middle button on a special cell to put the module in interactive mode, and press it again to exit interactive mode.</li>
+						<ul>
+							<li>Red: Using the maze below, navigate the defuser to the coordinates on the bottom left. The current position is in the middle of the cell.</li>
+							<li>Yellow: The coordinates in the middle represents a cell that was displayed on an ealier stage. Using the arrows, input the walls of that cell.</li>
+							<li>Blue: You are given walls and coordinates, one of which are missing. Replace the coordinate so that the walls match the coordinates.</li>
+							<li>Magenta: The defuser will have to find the exit of a normal 5x5 maze on the module.</li>
+							<li>Cyan: Same as magenta, but the maze is inverted (You can only walk through walls).</li>
+						</ul>
+					<li>Incorrect movement or answers, and ignoring special cells will strike.</li>
+			    </ul>
+				<img src="img/Forget Maze Not/Maze.svg" width="300" height="300" style="display:block;margin-left:auto;margin-right:auto;margin-top:20px;">
             </div>
             <div class="page-footer relative-footer">Page 1 of 1</div>
         </div>

```

Comment:
idk how to comment about pdf below, but pdfs are not required and autogenerated to minimize the size of the repo, unless pdf does not auto-generated properly.

Reply:
I apologize on missing that out beforehand.